src/tests/ftest/launch.py

```diff
@@ -1504,7 +1506,8 @@ def setup_test_files(self, yaml_dir, args):
             # Change the auto-storage extra yaml format if md_on_ssd is requested
             if args.nvme.startswith("auto_md_on_ssd"):
                 tier_0_type = "ram"
-                scm_size = 100
+                if scm_size != 0:
+                    scm_size = 100
```

Comment:
So even if passed in with, for example, `-t 50`, it gets overridden to 100?

Reply:
Limitation has been addressed